### ISSUE TITLE
Incomplete diagnostics when importing a non-existing module

### ISSUE THREAD

**Issue by saulecabrera** [AssemblyScript/assemblyscript#1334]
**Version** 0.10.0

**Current behavior**
When importing a non-existing module, the diagnostic is incomplete; it's missing the location. For the following unexisting import 

```ts
import * as foo from './bar';
```

the compilation phase reports:

```
ERROR: import assembly/bar not found
```

**Expected behavior**
For the case described above, the compilation phase should report complete diagnostics about the missing module:

```
ERROR TS{CODE}: Cannot find module './bar'

import * as foo from './bar';

in <file>.asc(line, column)
```

I found this while doing some tests for the language server.

**Comment by dcodeIO** [comment]
I've recently touched this in https://github.com/AssemblyScript/assemblyscript/pull/1307, which should be part of 0.10.1. Can you check if it's still an issue?

**Comment by saulecabrera** [comment]
@dcodeIO I can confirm that the issue is fixed in `0.10.1`, thanks!
